docs/src/main/sphinx/security/security-overview.rst

```diff
@@ -0,0 +1,160 @@
+=================
+Security overview
+=================
+
+After the initial :doc:`installation </installation>` of your cluster, security
+is the next major concern for successfully operating Trino. This overview
+provides an introduction to different aspects of configuring security for your
+Trino cluster.
+
+Aspects of configuring security
+-------------------------------
+
+The default installation of Trino has no security features enabled. Security
+can be enabled for different parts of the Trino architecture:
+
+* :ref:`security-client`
+* :ref:`security-inside-cluster`
+* :ref:`security-data-sources`
+
+Suggested configuration workflow
+--------------------------------
+
+To configure security for a new Trino cluster, follow this best practice
+order of steps. Do not skip or combine steps.
+
+1. **Enable HTTPS**
+
+* Work with your security team.
```

Comment:
Indent these, since they are the content for the numbered list items:
```
1. **Enable HTTPS**

   * Work with your security team.
```
In this case 3 spaces, since that's the width of the numbered list item (plus a space). Otherwise, we actually get three different numbered lists, which which isn't right semantics.

You can also use `#.` for the list items, rather than the exact numbers, which is more consistent with non-ordered list formatting and makes it easy to rearrange items if needed.

Reply:
All done as suggested. Thanks.